cookbooks/geomio/doc/geomio.md

```diff
@@ -20,44 +27,61 @@ repository by executing the command
 
      git clone https://bitbucket.org/geomio/geomio.git
 
-or you can download geomIO [here]. You will then need to add the geomIO
+or you can download geomIO [here](https://bitbucket.org/geomio/geomio/downloads/).
+You will then need to add the geomIO
 source folders to your MATLAB path by running the file located in
 `/path/to/geomio/installation/InstallGeomIO.m`. An extensive documentation for
-how to use geomIO can be found [here][1]. Among other things, it explains [how
-to generate drawings in Inkscape] that can be read in by geomIO, which
+how to use geomIO can be found [here](https://bitbucket.org/). Among other things, it explains [how
+to generate drawings in Inkscape](https://bitbucket.org/#drawing) that can be read in by geomIO, which
```

Comment:
This link cannot be right. Can you check the previous link?

Reply:
Ah I see, I was just clicking the links in the pdf manual and it redirected me there. I'm guessing if I was logged into bitbucket the links would work as intended. Me being lazy!